### PR TITLE
Issue 10

### PR DESCRIPTION
**Pull request by greg-hellings**
Addresses #10 and #8

### CODE REVIEW

File: molecule/openstack/create.yml [hunk 1]
```diff
@@ -2,128 +2,6 @@
...
-      wait_for:
-        port: "{{ ssh_port }}"
-        host: "{{ item.address }}"
-        search_regex: SSH
-        delay: 10
-      with_items: "{{ lookup('file', molecule_instance_config) \
-                   | molecule_from_yaml }}"
+  roles:
+    - role: oasis_roles.molecule_openstack_ci
+      molecule_openstack_ci_state: absent
```
Comment (tehsmyers) [inline molecule/openstack/create.yml]: github ate my last comment, so apologies if this is redundant...

`requirements.yml` needs to include the ci role.

```suggestion
      molecule_openstack_ci_state: present
```

...or just delete the line.

**Review by tehsmyers — CHANGES_REQUESTED** [review]

File: molecule/openstack/create.yml (hunk 1 above)
Comment (greg-hellings) [reply molecule/openstack/create.yml]: Oops!

**Review by tehsmyers — APPROVED** [review]
The openstack scenario is still broken, but as far as a code review goes this is all good otherwise. :)

I think this is all that's left:

```diff
diff --git a/molecule/openstack/molecule.yml b/molecule/openstack/molecule.yml
index 7481590..46bb0a4 100644
--- a/molecule/openstack/molecule.yml
+++ b/molecule/openstack/molecule.yml
@@ -1,5 +1,7 @@
 dependency:
   name: galaxy
+  options:
+    role-file: molecule/shared/requirements.yml
 driver:
   name: openstack
 lint:
```

**Comment by greg-hellings** [comment]
Turns out it needed a bit more than that - epel packages depend on packages pulled from elsewhere, and ci.m1.tiny doesn't have sufficient memory to run all the way through a complete yum transaction.

**Pull request merged**